Source/JavaScriptCore/runtime/ImplementationVisibility.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2019 Apple Inc. All Rights Reserved.
```

Comment:
Me thinks you're a time traveler!

```suggestion
 * Copyright (C) 2022 Apple Inc. All Rights Reserved.
```